charts/openshift-metering/values.yaml

```diff
@@ -4,6 +4,9 @@ global:
 unsupportedFeatures:
   enableHDFS: false
 
+disableOCPFeatures:
```

Comment:
I just realized this has some double negatives going on. Let's not use a nested field here, let's just have `disableOCPFeatures: false`